feat(windscribe): static ip permanent port forwarding instructions (#122)
qdm12/gluetun-wiki: setup/providers/windscribe.md

### VPN server port forwarding

> #### Ephemeral Port Forwarding

Reviewer: nit to satisfy the markdown linter

```suggestion
#### Ephemeral port forwarding

```

1. Follow the [Windscribe instructions](https://windscribe.com/support/article/37/what-is-ephemeral-port-forwarding-and-how-to-use-it)
1. In your container configuration, set `FIREWALL_VPN_INPUT_PORTS` to the port you have been assigned, for example: `FIREWALL_VPN_INPUT_PORTS=8099`

> #### Permanent Port Forwarding (Static IP)

Reviewer: ```suggestion
#### Permanent port forwarding (static IP)

```

> 1. You will need to follow the [custom provider guide](https://github.com/qdm12/gluetun-wiki/blob/main/setup/providers/custom.md).
> 2. In your container configuration, set `FIREWALL_VPN_INPUT_PORTS` to the port you have been assigned, for example: `FIREWALL_VPN_INPUT_PORTS=8099`
> 3. It's reccomended that you use the configuration file that Windscribe gives you for [OpenVPN](https://windscribe.com/getconfig/openvpn) or [WireGuard](https://windscribe.com/getconfig/wireguard).

Reviewer: ```suggestion
1. Follow the [custom provider setup instructions](https://github.com/qdm12/gluetun-wiki/blob/main/setup/providers/custom.md), using the [openvpn](https://windscribe.com/getconfig/openvpn) or [wireguard](https://windscribe.com/getconfig/wireguard) configuration file that Windscribe provides for static ips.
1. In your container configuration, set `FIREWALL_VPN_INPUT_PORTS` to the port you have been assigned, for example: `FIREWALL_VPN_INPUT_PORTS=8099`
```
